toolchain/semantics/testdata/function/call/fail_param_count.carbon

```diff
@@ -0,0 +1,154 @@
+// CHECK:STDOUT:   {kind: FunctionDeclaration, arg0: callable3},
+// CHECK:STDOUT:   {kind: BindName, arg0: str5, arg1: node20},
+// CHECK:STDOUT:   {kind: IntegerLiteral, arg0: int0, type: node2},
+// CHECK:STDOUT:   {kind: IntegerLiteral, arg0: int1, type: node2},
+// CHECK:STDOUT:   {kind: IntegerLiteral, arg0: int2, type: node2},
+// CHECK:STDOUT:   {kind: IntegerLiteral, arg0: int3, type: node2},
+// CHECK:STDOUT:   {kind: IntegerLiteral, arg0: int4, type: node2},
+// CHECK:STDOUT:   {kind: IntegerLiteral, arg0: int5, type: node2},
+// CHECK:STDOUT:   {kind: FunctionDefinition, arg0: node20, arg1: block0},
+// CHECK:STDOUT: ]
+// CHECK:STDOUT: node_blocks: [
+// CHECK:STDOUT:   [
+// CHECK:STDOUT:   ],
+// CHECK:STDOUT:   [
+// CHECK:STDOUT:     node5,
+// CHECK:STDOUT:     node6,
+// CHECK:STDOUT:     node7,
+// CHECK:STDOUT:     node10,
+// CHECK:STDOUT:     node11,
+// CHECK:STDOUT:     node12,
+// CHECK:STDOUT:     node17,
+// CHECK:STDOUT:     node18,
+// CHECK:STDOUT:     node19,
+// CHECK:STDOUT:     node20,
+// CHECK:STDOUT:     node21,
+// CHECK:STDOUT:     node28,
+// CHECK:STDOUT:   ],
+// CHECK:STDOUT:   [
+// CHECK:STDOUT:     node8,
+// CHECK:STDOUT:     node9,
+// CHECK:STDOUT:   ],
+// CHECK:STDOUT:   [
+// CHECK:STDOUT:     node9,
+// CHECK:STDOUT:   ],
+// CHECK:STDOUT:   [
+// CHECK:STDOUT:     node13,
+// CHECK:STDOUT:     node14,
+// CHECK:STDOUT:     node15,
+// CHECK:STDOUT:     node16,
+// CHECK:STDOUT:   ],
+// CHECK:STDOUT:   [
+// CHECK:STDOUT:     node14,
+// CHECK:STDOUT:     node16,
+// CHECK:STDOUT:   ],
+// CHECK:STDOUT:   [
+// CHECK:STDOUT:     node22,
+// CHECK:STDOUT:   ],
+// CHECK:STDOUT:   [
+// CHECK:STDOUT:     node22,
+// CHECK:STDOUT:   ],
+// CHECK:STDOUT:   [
+// CHECK:STDOUT:     node23,
+// CHECK:STDOUT:     node24,
+// CHECK:STDOUT:   ],
+// CHECK:STDOUT:   [
+// CHECK:STDOUT:     node23,
+// CHECK:STDOUT:     node24,
+// CHECK:STDOUT:   ],
+// CHECK:STDOUT:   [
+// CHECK:STDOUT:     node25,
+// CHECK:STDOUT:     node26,
+// CHECK:STDOUT:   ],
+// CHECK:STDOUT:   [
+// CHECK:STDOUT:     node25,
+// CHECK:STDOUT:     node26,
+// CHECK:STDOUT:   ],
+// CHECK:STDOUT:   [
+// CHECK:STDOUT:     node27,
+// CHECK:STDOUT:   ],
+// CHECK:STDOUT:   [
+// CHECK:STDOUT:     node27,
+// CHECK:STDOUT:   ],
+// CHECK:STDOUT: ]
+
+fn Run0() {}
+fn Run1(a: i32) {}
+// CHECK:STDERR: {{.*}}/toolchain/semantics/testdata/function/call/fail_param_count.carbon:[[@LINE+2]]:9: Redefining a in the same scope.
+// CHECK:STDERR: {{.*}}/toolchain/semantics/testdata/function/call/fail_param_count.carbon:[[@LINE-2]]:9: Previous definition is here.
+fn Run2(a: i32, b: i32) {}
+
+fn Main() {
+  // CHECK:STDERR: {{.*}}/toolchain/semantics/testdata/function/call/fail_param_count.carbon:[[@LINE+2]]:7: No matching callable was found.
+  // CHECK:STDERR: {{.*}}/toolchain/semantics/testdata/function/call/fail_param_count.carbon:[[@LINE-8]]:1: Received 1 arguments, but require 0 parameters.
```

Comment:
Do we have a good way to get the right suffix here ("argument" versus "arguments")?

Reply:
Not right now AFAIK